abr_control/arms/tests/test_mujoco_config.py

```diff
@@ -0,0 +1,152 @@
+import numpy as np
+
+from abr_control.arms.mujoco_config import MujocoConfig as arm
+from abr_control.interfaces.mujoco import Mujoco
+
+from .dummy_mujoco_arm import TwoJoint
+
+
+# TODO
+# def test_connect
+
+
+# TODO
+# def test_load_state
+
+
+def test_g():
+    test_arm = TwoJoint()
+    # set up the Mujoco interface and config
+    robot_config = arm('twojoint')
+    interface = Mujoco(robot_config=robot_config)
+    interface.connect()
+
+    q_vals = np.linspace(0, 2*np.pi, 50)
+    for q0 in q_vals:
+        for q1 in q_vals:
+            q = [q0, q1]
+            assert np.allclose(robot_config.g(q), test_arm.g(q))
+
+
+# TODO
+# def test_dJ():
+
+
+def test_J():
+    test_arm = TwoJoint(L0=.2, L1=.4)
+    # set up the Mujoco interface and config
+    robot_config = arm('twojoint')
+    interface = Mujoco(robot_config=robot_config)
+    interface.connect()
+
+    q_vals = np.linspace(0, 2*np.pi, 50)
+    for q0 in q_vals:
+        for q1 in q_vals:
+            q = [q0, q1]
+            print('q: ', q)
+            assert np.allclose(
+                robot_config.J('link0', q, object_type='geom'),
+                test_arm.J_link0(q))
+            assert np.allclose(
+                robot_config.J('link1', q, object_type='geom'),
+                test_arm.J_link1(q))
+            assert np.allclose(
+                robot_config.J('link2', q, object_type='geom'),
+                test_arm.J_link2(q))
+            print('J mujoco: ', robot_config.J('EE', q))
+            print('J test: ', test_arm.J_EE(q))
+            assert np.allclose(
+                robot_config.J('EE', q), test_arm.J_EE(q))
+
+
+def test_M():
+    test_arm = TwoJoint()
+    robot_config = arm.Config()
+
+    q_vals = np.linspace(0, 2*np.pi, 50)
+    for q0 in q_vals:
+        for q1 in q_vals:
+            q = [q0, q1]
+            assert np.allclose(robot_config.M(q), test_arm.M(q))
+
+
+def test_R():
```

Comment:
currently fails, need to add ability to `def R()` to accept `object_type` like we have in `def Tx()` if it is possible, or remove the joint check for this test

Reply:
There doesn't seem to be an API function for getting the `joint` orientation, only `body` objects

Reply:
fixed!